addon/components/o-s-s/toggle-buttons.stories.js

```diff
@@ -0,0 +1,69 @@
+import hbs from 'htmlbars-inline-precompile';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/OSS::ToggleButtons',
+  component: 'upload-item',
```

Comment:
copy-pasta issue :p

Reply:
Fixed 🙃 in https://github.com/upfluence/oss-components/commit/58f8b2e7f8aa651fd5c6e4bcbaf3abf334763996